_automating-configurations/default-use-cases.md

```diff
@@ -0,0 +1,200 @@
+GET /my-nlp-index/_search
+{
+  "_source": {
+    "excludes": [
+      "passage_embedding"
+    ]
+  },
+  "query": {
+    "neural": {
+      "passage_embedding": {
+        "query_text": "Hi world",
+        "k": 100
+      }
+    }
+  }
+}
+```
+
+### Additional Flexibility:
+
+* The workflow we created with the above use case set up all the necessary resources for sample semantic search, you can view the resources created utilizing the get status API with the workflowID that was created: 
+    * `GET /_plugins/_flow_framework/workflow/8xL8bowB8y25Tqfenm50/_status`
+
+### Semantic Search Pre Defined defaults:
+
+* Each use case has out of the box defaults based on sample configurations, however any of the given defaults can be overwritten by the user through the request body. 
+
+```
+{
+    "template.name": "semantic search with cohere embedding",
+    "template.description": "Setting up semantic search, with a Cohere embedding model",
+    "create_connector.name": "cohere-embedding-connector",
+    "create_connector.description": "The connector to Cohere's public embed API",
+    "create_connector.protocol": "http",
+    "create_connector.model": "embed-english-v3.0",
+    "create_connector.input_type": "search_document",
+    "create_connector.truncate": "end",
+    "create_connector.credential.key": "123",
+    "create_connector.actions.url": "https://api.cohere.ai/v1/embed",
+    "create_connector.actions.request_body": "{ \"texts\": ${parameters.texts}, \"truncate\": \"${parameters.truncate}\", \"model\": \"${parameters.model}\", \"input_type\": \"${parameters.input_type}\" }",
+    "create_connector.actions.pre_process_function": "connector.pre_process.cohere.embedding",
+    "create_connector.actions.post_process_function": "connector.post_process.cohere.embedding",
+    "register_remote_model.name": "Cohere english embed model",
+    "register_remote_model.description": "cohere-embedding-model",
+    "create_ingest_pipeline.pipeline_id": "nlp-ingest-pipeline",
+    "create_ingest_pipeline.description": "A text embedding pipeline",
+    "text_embedding.field_map.input": "passage_text",
+    "text_embedding.field_map.output": "passage_embedding",
+    "create_index.name": "my-nlp-index",
+    "create_index.settings.number_of_shards": "2",
+    "create_index.mappings.method.engine": "lucene",
+    "create_index.mappings.method.space_type": "l2",
+    "create_index.mappings.method.name": "hnsw",
+    "text_embedding.field_map.output.dimension": "1024",
+    "create_search_pipeline.pipeline_id": "default_model_pipeline"
+}
+```
+
+### Example overriding of params:
+
+```
+POST /_plugins/_flow_framework/workflow?use_case=semantic_search_with_cohere_embedding_query_enricher
+{
+    "create_connector.model" : "embed-multilingual-v3.0",
+    "create_ingest_pipeline.text_embedding.field_map.output": "book_embedding",
+    "create_index.name": "sparse-book-index"
+}
+```
+
+* In the example above, we are changing the Cohere model we want to use
```

Comment:
Duplicate mention of "example". Suggest to use in the title or description once, but not twice.

Reply:
This line shouldn't be a bullet point.